ChangeLog.md

```diff
@@ -4,29 +4,244 @@ Notable changes to the FreeOrion project will be documented in this file.
 
 
 
-## [v0.4.8] - TBD
+## [v0.4.8] - 2017-06-??
 
 
 ### Key Changes
 
+- Imperial stockpile
+    - Unallocated production is stored, and can be used on colonies that are disconnected from the empire supply network
+    - Withdrawl limits based on planet stockpile meters and indicated on map and production screen
```

Comment:
Withdrawal
And I'd suggest clarifying that the limit is global, perhaps
"Withdrawal limits are empire-wide, based on..."

Reply:
Corrected and suggestion incorporated.